projectional/src/main/java/jetbrains/jetpad/projectional/selection/SelectionSupport.java

```diff
@@ -338,6 +338,36 @@ public void run() {
     }
   }
 
+  private boolean cursorIsInSinglePossiblePos(Cell cell) {
```

Comment:
@timzam Some one-pisition cells like `(` in function call satisfy this condition because left position is not allowed to navigate to. Seems like "empty" is not good enough for cell having a text. What do you think?